docs/guide/combining-schemas.md

```diff
@@ -41,7 +41,7 @@ const ajv = new Ajv()
 const validate = ajv.addSchema(defsSchema).compile(schema)
 ```
 
-See [Options](./api.md#options) and [addSchema](./api.md#add-schema) method.
+See [Options](../options.md) and [addSchema](../api.md###ajv.addSchema) method.
```

Comment:
```suggestion
See [Options](../options.md) and [addSchema](../api.md#add-schema) method.
```

shouldn't it be this?

Reply:
It seems neither works, it probably should be `#ajv-addschema-schema-object-object-key-string-ajv`

Reply:
Oh, I see, api.md has `<a name="add-schema"></a>` and it has to be `<a id="add-schema"></a>` - could you please update it too in api.md, then my ref would work...

Reply:
updated!

Reply:
@epoberezkin let me know if any further changes need to be made from my end!

Reply:
@epoberezkin bump - just following up on this thank you